### PR TITLE
Hotfix/improving ado and gcp docs

### PR DESCRIPTION
**Pull request by matan84**
# Description

Added warning to GCP docs and extra mappings to ADO docs

## Updated docs pages

- GCP Mapping Extra Resources (`/build-your-software-catalog/sync-data-to-catalog/cloud-providers/gcp/examples/mapping_extra_resources`)
- ADO Examples (`/build-your-software-catalog/sync-data-to-catalog/git/azure-devops/examples`)

### CODE REVIEW

File: docs/build-your-software-catalog/sync-data-to-catalog/git/azure-devops/examples.md [hunk 1]
```diff
@@ -83,6 +85,10 @@ In the following example you will ingest your Azure Devops teams and their membe
 
 <MembersBlueprint/>
 
+<ProjectBlueprint>
```
Comment (hadar-co) [inline docs/build-your-software-catalog/sync-data-to-catalog/git/azure-devops/examples.md]: close tag

File: docs/build-your-software-catalog/sync-data-to-catalog/cloud-providers/gcp/examples/mapping_extra_resources.md [hunk 2]
```diff
@@ -43,6 +43,10 @@ Create an integration configuration for the resource. The integration configurat
 As you may see, some of the properties are digested using `.versioned_resources | max_by(.version).resource | <property_value>`. This is done, because the Cloud Assets contains all of the available versions of the resource within each inventory item, so that's why we take the most recent version of the asset in the inventory, and from that version we take the property value.
 All of the other properties, which are not versioned, are properties we digest directly from the asset in the Asset inventory, which are all listed [here](https://cloud.google.com/asset-inventory/docs/reference/rest/v1/Asset). These don't have multiple versions, and so are directly digested.
 
+:::warning
```
Comment (hadar-co) [inline docs/build-your-software-catalog/sync-data-to-catalog/cloud-providers/gcp/examples/mapping_extra_resources.md:46]: add a title to the admonition

**Review by hadar-co — CHANGES_REQUESTED** [review]

File: docs/build-your-software-catalog/sync-data-to-catalog/cloud-providers/gcp/examples/mapping_extra_resources.md (hunk 2 above)
Comment (matan84) [reply docs/build-your-software-catalog/sync-data-to-catalog/cloud-providers/gcp/examples/mapping_extra_resources.md:46]: Can you please direct me to an example?

Comment (hadar-co) [reply docs/build-your-software-catalog/sync-data-to-catalog/cloud-providers/gcp/examples/mapping_extra_resources.md:46]: :::warning title goes here
content here
:::

File: docs/build-your-software-catalog/sync-data-to-catalog/cloud-providers/gcp/examples/mapping_extra_resources.md [hunk 3]
```diff
@@ -43,6 +43,10 @@ Create an integration configuration for the resource. The integration configurat
 As you may see, some of the properties are digested using `.versioned_resources | max_by(.version).resource | <property_value>`. This is done, because the Cloud Assets contains all of the available versions of the resource within each inventory item, so that's why we take the most recent version of the asset in the inventory, and from that version we take the property value.
 All of the other properties, which are not versioned, are properties we digest directly from the asset in the Asset inventory, which are all listed [here](https://cloud.google.com/asset-inventory/docs/reference/rest/v1/Asset). These don't have multiple versions, and so are directly digested.
 
+:::warning Resources not digested from Asset Inventory
+   The cloudresourcemanager.googleapis.com/Folder, cloudresourcemanager.googleapis.com/Organization, cloudresourcemanager.googleapis.com/Project, and pubsub.googleapis.com/Topic are fetched directly from Google Cloud's API, not using the Asset Inventory. Therefore, access their properties directly, without using the jq command specified above.
```
Comment (hadar-co) [inline docs/build-your-software-catalog/sync-data-to-catalog/cloud-providers/gcp/examples/mapping_extra_resources.md:47]:
```suggestion
   The `cloudresourcemanager.googleapis.com/Folder`, `cloudresourcemanager.googleapis.com/Organization`, `cloudresourcemanager.googleapis.com/Project`, and `pubsub.googleapis.com/Topic` are fetched directly from Google Cloud's API, not using the Asset Inventory. Therefore, access their properties directly, without using the jq command specified above.
```

**Review by hadar-co — APPROVED** [review]

**Pull request merged**